admin-ui/src/component/my-units/MyUnitRecurringReservation/RecurringReservationDone.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import type { ErrorType } from "common/types/gql-types";
+import { Button } from "hds-react/components/Button";
+import { useTranslation } from "react-i18next";
+import styled from "styled-components";
+import { H1, H6 } from "common/src/common/typography";
+import { ActionsWrapper } from "./commonStyling";
+import { ReservationList } from "./ReservationsList";
+import { useNotification } from "../../../context/NotificationContext";
+
+const InfoSection = styled.p`
+  margin-bottom: 2rem;
+  margin-top: 0;
+`;
+
+const StyledH6 = styled(H6)`
+  margin-bottom: 0;
+`;
+
+export type ReservationsMade = Array<{
+  reservationPk?: number;
+  startTime: string;
+  endTime: string;
+  date: Date;
+  error?: string | ErrorType[];
+}>;
```

Comment:
Is there motivation for the type format? Defining it as an object and referring to it as `ReservationsMade[]` would be preferable.